docs/Release_notes.md

```diff
@@ -1,6 +1,17 @@
 Jpo-s3-deposit Release Notes
 ----------------------------
 
+Version 1.5.0, released June 2024
+----------------------------------------
+### **Summary**
+The changes for the jpo-s3-deposit 1.5.0 release include updated hard-coded kafka properties to be configurable, added support for GCS BLOB storage & an updated java version for the dev container.
```

Comment:
Lets capitalize Kafka in these notes. There are a few occurrences.

Reply:
Capitalized!

Reply:
Capitalized!